code/game/objects/structures/watercloset.dm

```diff
@@ -385,6 +392,12 @@
 			else if(H.w_uniform && wash_obj(H.w_uniform))
 				H.update_inv_w_uniform()
 
+			if(check_clothes(H))
+				SEND_SIGNAL(L, COMSIG_ADD_MOOD_EVENT, "badshower", /datum/mood_event/idiot_shower)
+			else
+				SEND_SIGNAL(L, COMSIG_ADD_MOOD_EVENT, "shower", /datum/mood_event/nice_shower)
+				H.set_hygiene(HYGIENE_LEVEL_CLEAN)
```

Comment:
Make showering with your clothes clean you too, the mood punishment is already fair enough

Reply:
no else people will still shower with their clothes on

Reply:
Make the mood worse then

Reply:
no